action.yml

```diff
@@ -162,17 +174,9 @@ runs:
         GH_PR_EVENT_NUMBER: ${{ inputs.pr_event_number }}
         CLI_VERSION_TAG: ${{ steps.build_vars.outputs.cli_version }}
         RECIPE_PATH: ${{ steps.build_vars.outputs.recipe_path }}
+        RUST_LOG_STYLE: always
+        CLICOLOR_FORCE: '1'
       run: |
-        podman run \
-          -v buildah-imagestores:/usr/lib/containers/storage \
-          -v buildah-graphroot:/var/lib/containers/storage \
-          -v buildah-runroot:/run/containers/storage \
-          -v $PWD:/bluebuild \
-          --env-host \
-          --network=host \
-          --privileged \
-          --device /dev/fuse \
-          ghcr.io/blue-build/cli:${CLI_VERSION_TAG}-alpine \
-          build -v -B buildah --squash --push ${RECIPE_PATH} \
+        bluebuild build -v -B podman --squash --push ${RECIPE_PATH} \
```

Comment:
Could these two steps be combined into one, only making `-B podman` based on the value? Maybe using some GHA templating?

Reply:
Also using full CLI flag names in CI would be preferred for better code readability.